_source/handbook/02_drive.md

```diff
@@ -253,6 +253,20 @@ To accomplish this, just annotate those asset elements with `data-turbo-track="r
 </head>
 ```
 
+## Removing Assets When They Change
+
+As we saw above, Turbo Drive merges the contents of the `<head>` elements. When a page depends on CSS or JavaScript that other pages do not, it can be useful to remove them when navigating away from the page.
+
+Rendering a `<script>`, `<link>`, or `<style>` element with `data-turbo-track="reload"` instructs Turbo Drive to dynamically remove the element when it is no absent from a navigation's response.
```

Comment:
`<script>` tags are an odd case here because even if you remove them from the DOM the JS context won't change and the evaluated JS code keeps running in the browser.

@seanpdoyle I think we could focus the documentation on styles and explain that the same approach doesn't work with scripts.

Reply:
"no absent" or "not absent" 🤔